entity-types/infra-awsmqexchange/definition.yml

```diff
@@ -0,0 +1,7 @@
+domain: INFRA
+type: AWSMQEXCHANGE
```

Comment:
A couple of questions regarding this.

a)
Do you have an approval from Infra for this new entity type ?
There are already several MQ* types for AWS.

b)
Are they going to index it ?

Asking because this entity type does not have any [synthesis rules](https://github.com/newrelic/entity-definitions/blob/main/docs/entities/synthesis.md). So a team would need to send over entity indexation messages from their systems to EP.